Client/Frontend/Browser/Tab.swift

```diff
@@ -387,9 +386,8 @@ class Tab: NSObject {
         DispatchQueue.main.async {
             webView.evaluateJavaScript(getHtmlToStringJSCall, completionHandler: { html, _ in
                 guard let htmlString = html as? String else { return }
-                rewards.reportLoadedPage(url: url, tabId: tabId, html: htmlString) { verified in
-                    self.tabDelegate?.tab(self, isVerifiedPublisher: verified)
-                }
+                let faviconURL = URL(string: self.displayFavicon?.url ?? "")
```

Comment:
I would add a log here in case favicon can't be parsed to `URL`

Reply:
What if there's just no favicon for this given tab?

Reply:
For default it a website's url is set from what I see in `setIconURL`
`let defaults = defaultFavicon(url)`

Reply:
Wouldn't that log an error for all Tab's using that default favicon then? Since theres no URL?

Reply:
I did some tests and for default favicons it just appends `/favicon.ico` to the url.

Example url without favicon `https://www.suchedniow.pl/`

```
// If we haven't got any options icons, then use the default at the root of the domain.
                if let url = NSURL(string: "/favicon.ico", relativeTo: url as URL), icons.isEmpty, let absoluteString = url.absoluteString {
                    let icon = Favicon(url: absoluteString)
                    icons = [icon]
                }
```

The only place where there's no faviconUrl is on new tab, reader mode website have faviconUrl as well.

I guess it's not a big deal to add or don't add those logs, we can always remove it after

Reply:
Gotcha, thanks for checking into it :) I'll add the logs and re-request review after